File: 백준/Bronze/1152. 단어의 개수/단어의 개수.c
#include <stdio.h>
#include <string.h>



int main(void) {
	char arr[1000001];
	scanf("%[^\n]", arr);
	int cnt = 0, len = strlen(arr);
	if (arr[0] == ' ' && len == 1) {
		printf("%d", cnt);
		return 0;
	}
	for (int i = 1; i < len - 1; i++) {
		if (arr[i] == ' ') cnt++;
	}
	printf("%d", cnt + 1);
	
	return 0; 
}

File: 백준/Bronze/1152. 단어의 개수/단어의 개수.py
arr = input()
cnt = 0
if(arr[0] == ' ' and len(arr) == 1):
    print(cnt)
    exit()
for i in range(1, len(arr) - 1):
    if(arr[i] == ' '):
        cnt += 1
print(cnt + 1)